task_manager/settings.py

```diff
@@ -107,14 +106,8 @@
     }
 }
 
-if not DEBUG:
-    DATABASES = {
-        "default": dj_database_url.config(
-            default=os.getenv("DATABASE_URL"),
-            conn_max_age=600,
-            conn_health_checks=True,
-        ),
-    }
+db_from_env = dj_database_url.config(conn_max_age=600)
+DATABASES["default"].update(db_from_env)
```

Comment:
will it allow the application to start locally without a dockerized setup?

Reply:
it will. I checked.

Reply:
yes, it will. I checked. if there is a postgres db url in env file, the app connects with postgres. Otherwise it creates a sqlite3 file.